test/lockstep_gen_SUITE.erl

```diff
@@ -238,7 +238,7 @@ timeout(Config) ->
         new_connection ->
             [{count, 2}] = ets:lookup(Tid, count),
             loop ! close
-    after 100 ->
+    after 1000 ->
```

Comment:
The idle timeout is set to `500` for the test, so it's not valid to expect a second connection 100ms after the first.

Reply:
Nice.